### PR TITLE
Fix window borders

### PR DESCRIPTION
**Pull request by crondog**
This fixes floating borders when moving outside of the output. Not sure what
happens with dual screen as I only have 1 monitor

### CODE REVIEW

**Comment by SirCmpwn** [comment]
LGTM. Will let @mikkeloscar merge though

**Comment by mikkeloscar** [comment]
@crondog this is great. You just need to handle the titlebar (blue bg + lighter blue top/bottom line) the same as you handle the text. Right now they are moved down if you move the window past the top edge.

Also the bottom border seem to be cut off one border_thickness too early, try to use 10 pixel border to see this more clearly:
sway_border
(bottom edge of the output)

**Comment by crondog** [comment]
@mikkeloscar good catch. I was testing this with my terminal which ended up covering the titlebar thus making it look like it worked

File: sway/layout.c [hunk 1]
```diff
@@ -374,25 +374,50 @@ void move_workspace_to(swayc_t* workspace, swayc_t* destination) {
...
+	else if (g->origin.x + g->size.w - right > res->w) {
+		g->size.w = res->w - g->origin.x + right;
+	}
+	
+	g->size.h += top + bottom;
+	if (g->origin.y - top < 0) {
+		g->size.h += g->origin.y - top;
+	}
+	else if (g->origin.y + g->size.h - top > res->h) {
+		g->size.h = res->h - g->origin.y + top ;
```
Comment (mikkeloscar) [inline sway/layout.c]: ugly space before ';' :)

**Comment by mikkeloscar** [comment]
@crondog great, will merge when build passes.

**Comment by crondog** [comment]
Woo hoo

**Pull request merged**